pkg/generator/helper/testdata/TestProcessMetaFiles/custom_step_code_generated.golden

```diff
@@ -200,6 +201,27 @@ func testStepMetadata() config.StepData {
 					},
 				},
 			},
+			Outputs: config.StepOutputs{
+				Resources: []config.StepResources{
+					{
+						Name: "commonPipelineEnvironment",
+						Type: "piperEnvironment",
+						Parameters: []map[string]interface{}{
```

Comment:
wouldn't this be `[]map[string]string`?

Reply:
`Parameters` is defined as map[string]interface{} type

Reply:
ok, grrrr ...